src/typescript/mit-open-api-axios/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@mitodl/open-api-axios",
+  "version": "0.0.0",
+  "description": "Library for requesting data from mit-open APIs",
+  "main": "dist/index.js",
```

Comment:
I think we should add an `exports` section:
```
"exports": {
    ".": "./dist/index.js",
    "./v1": "./dist/v1/index.js",
}
```
which would allow imports like
```ts
import { v1 } from "@mitodl/open-api-axios"

// import specifically from v1
import * as otherV1 from "@mitodl/open-api-axios/v1"
import { LearningResourcesSearchApi } from "@mitodl/open-api-axios/v1"
import type { CourseResource } from "@mitodl/open-api-axios/v1"
``` 

See also https://nodejs.org/api/packages.html#package-entry-points

_(We could also use only `exports`, I think)_